packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx

```diff
@@ -132,7 +132,6 @@ function CategoryList({
           overflowY: 'auto',
           willChange: 'transform',
           padding: '5px 0',
-          ...(!embedded && { maxHeight: 175 }),
```

Comment:
I think removing this is probably a bit too large, maybe doubling it or scaling based on viewport is a better way to go?